### DISCUSSION TITLE
Can we disable some built-in functions?

### DISCUSSION THREAD

**Discussion by purplefox** [antonmedv/expr#358] [General]
Hi,

I am looking at using expr in my new project (A new event streaming engine). So far it looks great!

I'd like to use expr to implement expressions for the language that we expose to the end user. I can see that expr has some built-in functions, but we'd like to have full control over what functions are exposed in the language. Is there any way to disable some of the built-in functions?

**Comment by antonmedv** [comment]
Currently not possible, be maybe we can add an option like expr.Disable(‘…’)

**Comment by PranavPeshwe** [comment]
I wonder whether we can alias built-in functions by declaring our own
functions by the same name(s). Our functions could be no-ops.
